### ISSUE TITLE
Some new files to be excluded where found.

### ISSUE THREAD

**Issue by liviuconcioiu** [phpmyadmin/phpmyadmin#19290]
https://github.com/phpmyadmin/phpmyadmin/actions/runs/10932009994/job/30348076519#step:13:23

```
Run ./scripts/check-release-excludes.sh release/phpMyAdmin-5.2+snapshot-all-languages.tar.gz
Searching for files
Found: phpMyAdmin-5.2+snapshot-all-languages/vendor/pragmarx/google2fa/.github/
Found: phpMyAdmin-5.2+snapshot-all-languages/vendor/pragmarx/google2fa/.github/workflows/
Found unexpected file: vendor/pragmarx/google2fa/.github/workflows/run-tests.yml with extension yml
Found: phpMyAdmin-5.2+snapshot-all-languages/vendor/pragmarx/google2fa/.github/workflows/run-tests.yml
Some new files to be excluded where found.
Please update create-release.sh
Error: Process completed with exit code 1.
```

https://github.com/phpmyadmin/phpmyadmin/actions/runs/10932009994/job/30348076727#step:12:28

```
Run ./bin/internal/check-release-excludes.sh release/phpMyAdmin-6.0+snapshot-all-languages.tar.gz
Searching for files
Found: phpMyAdmin-6.0+snapshot-all-languages/vendor/pragmarx/google2fa/.github/
Found: phpMyAdmin-6.0+snapshot-all-languages/vendor/pragmarx/google2fa/.github/workflows/
Found unexpected file: vendor/pragmarx/google2fa/.github/workflows/run-tests.yml with extension yml
Found: phpMyAdmin-6.0+snapshot-all-languages/vendor/pragmarx/google2fa/.github/workflows/run-tests.yml
Found unexpected file: vendor/psr/event-dispatcher/.editorconfig with extension editorconfig
Found: phpMyAdmin-6.0+snapshot-all-languages/vendor/psr/event-dispatcher/.editorconfig
Found unexpected file: vendor/spomky-labs/cbor-php/SECURITY.md with extension md
Found unexpected file: vendor/spomky-labs/pki-framework/SECURITY.md with extension md
Found unexpected file: vendor/web-auth/cose-lib/SECURITY.md with extension md
Some new files to be excluded were found.
Please update create-release.sh
Error: Process completed with exit code 1.
```

**Comment by williamdes** [comment]
Thank you for reporting this!

**Comment by MauricioFauth** [comment]
Fixed by ec54e14c1cf6a0976acc5ca090a1c884c4aa4cdb and 7228f9807d3c5f0c0a8100eaf3450a373b939e6f.
